services/powchain/src/test/java/tech/pegasys/teku/services/powchain/PowchainServiceTest.java

```diff
@@ -39,11 +40,13 @@
 import tech.pegasys.teku.service.serviceutils.ServiceConfig;
 import tech.pegasys.teku.spec.Spec;
 import tech.pegasys.teku.spec.TestSpecFactory;
+import tech.pegasys.teku.spec.datastructures.state.beaconstate.BeaconState;
 import tech.pegasys.teku.storage.api.Eth1DepositStorageChannel;
 
 public class PowchainServiceTest {
```

Comment:
I'd test something new here. At least that we start preElectra and don't start postElectra

Reply:
Good point actually. Added a couple of relevant tests. Changed the other tests to use `.start()` as well to cover the preElectra cases.